tools/mksurfdata_map/src/mksurfdat.F90

```diff
@@ -1159,16 +1165,26 @@ program mksurfdat
                 call abort()
              end if
           end if
+          
+          
+          ! IV Read input lake pct data
+          read(nfdynlak, '(A195,1x,I4)', iostat=ier) string, year
```

Comment:
Read this in as year2, and then compare to year from above and abort if they aren't the same. This is similar to the abort for harvest above.

Reply:
- [ ] I actually wonder if we want somewhat more rework than just what @ekluzek suggested above. Specifically: the code as written assumes separate files for the year-by-year lake data vs. the year-by-year pft and harvest data. Would it make sense to put these all on the same file? (I feel like the most straightforward thing would be to change the format of this file slightly so that there is one line per year, with format: 'year pftfilename harvestfilename lakefilename'. However, that's more a comment for @ekluzek because I don't expect you (@Ivanderkelen) to do that rework unless you're actively excited about doing it. And I'm fine with sticking with the current format of having separate lines for each file, with multiple lines per year, for now.)

- [ ] We'll also need changes in mksurfdata.pl to add the names of the dynamic lake files to the year-by-year text file(s).

How exactly we handle this will depend on whether we're going to put transient lake areas on all of our landuse timeseries files moving forward. @ekluzek do you feel we should just go ahead and do that, assuming we have the data? I'm thinking yes, but do you have thoughts on it?